### PR TITLE
feat(sum): sum by 'row' or 'column'

### PR DESCRIPTION
**Pull request by JeffersonH44**

### CODE REVIEW

**Comment by targos** [comment]
Can you add/run tests? You will see that it doesn't work :)
You have to `require('./matrix')` instead of abstractMatrix

**Comment by JeffersonH44** [comment]
should I add those test on dynamicMethods.js?

**Comment by targos** [comment]
no, you can create a new file (sum.js)

**Comment by targos** [comment]
similar to https://github.com/mljs/matrix/blob/master/test/matrix/minMax.js

File: src/util.js [hunk 1]
```diff
@@ -108,3 +110,33 @@ exports.getRange = function getRange(from, to) {
     }
     return arr;
 };
+
+exports.sumByRow = function sumByRow(matrix) {
+    var sum = Matrix.Matrix.zeros(matrix.rows, 1);
+    for (var i = 0; i < matrix.rows; ++i) {
+        for (var j = 0; j < matrix.columns; ++j) {
+            sum[i][0] += matrix[i][j];
```
Comment (targos) [inline src/util.js]: Please use `sum.set()` and `matrix.get()`.
We abstract out the fact that Matrix is a 2D array to allow different kinds of storages (like all the views)

File: src/util.js [hunk 2]
```diff
@@ -108,3 +110,33 @@ exports.getRange = function getRange(from, to) {
...
+        }
+    }
+    return sum;
+};
+
+exports.sumByColumn = function sumByColumn(matrix) {
+    var sum = Matrix.Matrix.zeros(1, matrix.columns);
+    for (var i = 0; i < matrix.rows; ++i) {
+        for (var j = 0; j < matrix.columns; ++j) {
+            sum[0][j] += matrix[i][j];
```
Comment (targos) [inline src/util.js]: same

**Comment by targos** [comment]
Thanks!

**Pull request merged**